cmd/pw_ws_broker/main.go

```diff
@@ -148,13 +144,13 @@ func run(c *cli.Context) error {
 	lowRoute := c.String("route-key-low")
 	highRoute := c.String("route-key-high")
 
-	hasRedis := flagWasSet("redis", c.App.Flags)
-	hasNats := flagWasSet("nats", c.App.Flags)
-	hasRabbit := flagWasSet("rabbitmq", c.App.Flags)
+	hasRedis := "" != redis
+	hasNats := "" != nats
+	hasRabbit := "" != rabbitmq
```

Comment:
Is there any method further down to actually validate that it's a correct format?

Doesn't look to catch that the format is wrong and just blindly tries to make the provided input fit.

Reply:
It won't connect. and since we need to connect before proceeding, we get a nice error message and the program exits